docs/data-collection/tear-down.md

```diff
@@ -38,25 +42,37 @@
 ### ET arm, ET cables, infrared mirror and stimuli screen
 
 - [ ] Unplug the two cables (signal and power) connected to the ET arm.
-- [ ] Roll the cables and put them in the cupboard inside the Scanning room.
+- [ ] Roll the ET's cables and put them in the cupboard inside the Scanning room.
 - [ ] Remove the mirror frame from its rails mounted on the head coil and lay it on the bed.
 - [ ] Put the gloves on and cover the infrared mirror for storage.
 
     !!! danger "The infrared mirror MUST be manipulated with clean gloves at all times."
 
 - [ ] Take the projector's screen off and store it in its designated shelf.
-- [ ] Disconnect the ECG cable from the filter of the access panel, roll the cable.
-- [ ] Roll the RB tube and the GA tube and store them in the cupboard.
+
+### Gas mask/cannula and tubing, RB and tubing, ECG cables
+
+- [ ] Roll the ECG cable after checking the leads are bagged.
+- [ ] Bag the ECG cable once rolled.
+- [ ] Detach the RB tube from the RB inlet.
+- [ ] Roll the RB tube and the GA tube and store them bagged in the cupboard.
 
 ### Readying the scanner for the next session
 
-!!! warning "Every setting altered for the experiment MUST be put back to its original status at the end of the experiment (e.g., position of the bed, coil, emergency button, padding elements, etc.)"
+!!! important "Reset all scanner settings in the Scanning Room to defaults"
+
+    Every setting altered for the experiment MUST be put back to its original status at the end of the experiment (e.g., position of the bed, coil, emergency button, padding elements, etc.)
 
-- [ ] Put the used bed-sheet and the blanket inside the soiled linen bag.
+- [ ] Remove used blankets and bed-sheets <mark>**ONE-BY-ONE**</mark>:
+    - [ ] extend them <mark>**ONE-BY-ONE**</mark> to let any forgotten items fall on the floor before you fold it; and
+    - [ ] dispose of them <mark>**ONE-BY-ONE**</mark> in the adequate bin (soiled linen bag if they are fabric and trash if they are disposable).
 
-    !!! important "Put them away in the trash if they are disposable"
+    ??? danger "Make sure you do not accidentally dispose of valuable items or instrumentation"
 
-- [ ] Dispose all single-use sanitary protections.
+        At the end of the scanning session everyone is tired and the likelihood of a misshapen is large because the attention levels are at at their lowest.
```

Comment:
```suggestion
        At the end of the scanning session everyone is tired and the likelihood of a misshapen is large because the attention levels are at their lowest.
```